packages/css-engine/src/core/css-engine.test.ts

```diff
@@ -408,7 +479,9 @@ describe("CssEngine", () => {
     rule.styleMap.delete("display");
     expect(engine.cssText).toMatchInlineSnapshot(`
       "@media all {
-        .c { background-image: url(foo.png) }
+        .c {
+          background-image: url(foo.png)
```

Comment:
This means our published site will have less optimized css. Maybe use method instead of getter with some "pretty" flag?

Reply:
yes but only spaces, which compress well over gzip, we actually need to optimize by converting to shorthands and such or by using atomic css, for now I think it's not worth adding that flag that can only remove some whitespaces